packs/feat-effects/effect-emblazon-armament-shield.json

```diff
@@ -0,0 +1,57 @@
+{
+    "_id": "o04gwnrVtzWyIEs8",
+    "img": "systems/pf2e/icons/spells/dancing-shield.webp",
+    "name": "Effect: Emblazon Armament (Shield)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Emblazon Armament] when applied to a shield.</p>\n<ul>\n<li><strong>Shield</strong> The shield gains a +1 status bonus to its Hardness.</li>\n</ul>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 2
+        },
+        "rules": [
+            {
+                "choices": {
+                    "ownedItems": true,
+                    "types": [
+                        "armor"
+                    ]
+                },
+                "flag": "effectEmblazonArmamentShield",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.Armor"
+            },
+            {
+                "itemType": "armor",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "item:id:{item|flags.pf2e.rulesSelections.effectEmblazonArmamentShield}"
+                ],
+                "property": "hardness",
+                "type": "status",
```

Comment:
?

Reply:
Removed type status from effect, as not valid field in ItemAlteration